torch/_inductor/codegen/cuda/cuda_env.py

```diff
@@ -0,0 +1,31 @@
+import functools
+import logging
+from subprocess import PIPE, Popen
+
+import torch
+
+from ... import config
+
+
+def get_cuda_arch() -> str:
+    try:
+        cuda_arch = config.cuda.arch
+        if cuda_arch is None:
+            # Get Compute Capability of the first Visible device
+            major, minor = torch.cuda.get_device_capability(0)
+            cuda_arch = major * 10 + minor
```

Comment:
this appears to be an int or some kind of number, not a str as in the type hint, right?

Reply:
Updated with correct type hints.